platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyNotationIntegrationSpec.groovy

```diff
@@ -27,14 +27,14 @@ class DependencyNotationIntegrationSpec extends AbstractIntegrationSpec {
     def "understands dependency notations"() {
         when:
         buildFile <<  """
-import org.gradle.api.internal.artifacts.dependencies.*
+import org.gradle.api.internal.artifacts.dependencies.DefaultFileCollectionDependency
 configurations {
     conf
     gradleStuff
     allowsCollections
 }
 
-def someDependency = new DefaultSelfResolvingDependency(files('foo.txt'))
+def someDependency = new DefaultFileCollectionDependency(files('foo.txt'))
```

Comment:
🤔 Is this really supposed to be a directly created self resolving dependency or can it just be `files(...)`?

Reply:
This does not need to be directly created.